README.md

```diff
@@ -156,7 +157,7 @@ To make sure everything works well, run `docker ps`. You should see 4 containers
      4.3 If you wish to stop producing data, you can stop the containers using `docker-compose stop datagen` or `docker-compose stop producer`
 
 5. Check if the consumer shows the stream output successfully:
-  run `docker logs kafka-consumer -f` or `docker-compose logs -f consumer`.
+  run `docker-compose logs kafka-consumer -f` or `docker-compose logs -f consumer`.
```

Comment:
Not true. either docker logs kafka-consumer or docker-compose logs consumer. If you are using docker-compose, than it is by service name. If docker - it is by container name.